src/main/resources/static/js/knox.js

```diff
@@ -557,6 +557,10 @@ $('#apply-operators-tooltip').click(() => {
         case endpoint.operators.MERGE:
           endpoint.designSpaceMerge(inputSpaces, outputSpace, tolerance);
           break;
+
+        case endpoint.operators.REVERSE:
+          endpoint.designSpaceReverse(inputSpaces, outputSpace);
```

Comment:
I'd prefer that we keep the plurality of the nouns constant. The user should only be able to enter one inputSpace, so this variable name is misleading and confusing.

Reply:
I will change it.